### ISSUE TITLE
How does this work align (or not) with existing 18F frontend recommendations

### ISSUE THREAD

**Issue by meiqimichelle** [18F/united#4]
_For discussion_

One of the outcomes we'd like to see as a front end community at 18F is being able to [share code faster so that it is easier to jump between projects](https://github.com/18F/frontend/wiki). We have a front end guide with various recommendations (for example, [here's the section on naming classes](https://pages.18f.gov/frontend/#css-naming) ) . How does this work align with those recommendations? Should it? An answer could be "no", but we should talk about it :)

**Comment by thisisdano** [comment]
Yeah, I think of this work as a kind of rosetta stone, or universal language shared between different traditional taxonomic systems. It definitely fits into the code-sharing idea, and works at something of a different level from traditional "semantic" BEM-ish, OOCSS-ish naming systems.

**Comment by thisisdano** [comment]
It's also a system that specifically addresses the presentational concerns that form a bridge between visual design and front-end design — potentially making it simpler to move from design to production.

**Comment by meiqimichelle** [comment]
The holy grail: `universal language shared between different traditional taxonomic systems` -- careful ye who enter here ;)

Before you build out the entire thing, how are you planning on testing your assumption that this `potentially making it simpler to move from design to production` ?

**Comment by thisisdano** [comment]
1) Seems like an enjoyable challenge, maybe? (The "wisdom" of the fool, potentially.)
2) Dunno! This is my personal and anecdotal experience, but it hasn't yet undergone the rigor of more comprehensive testing. I'd have to work with others for a testing plan, and I think it might come through future work with Marigold.

And, tbh, it is _nearly_ 100% built out (at least for its limited feature set). Though, clearly, it's only a product of my personal idiosyncratic use, and could use some input from others familiar with the problems it's attempting to solve.

**Comment by meiqimichelle** [comment]
Ah, got it! So maybe for now, you'll continue to use it and find it useful (or not). We'll see what comes of the Marigold research. And then go from there?

**Comment by thisisdano** [comment]
That's pretty much it. It's all essentially personal work that I developed to use for cloud.gov and felt could have broader utility. I don't have any design-to-code projects for the next month, so I'll probably reach out to any other devs who might want to think this through with me. I consider it a WIP, albeit one that's seen project use.
